### PR TITLE
feat: add ast to glossary

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Adding AST to docs glossary
https://github.com/influxdata/openapi/pull/428

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.3/reference/glossary.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ influxdb/v2.3/tags: [glossary]
...
 ## A
 
+### abstract syntax tree (AST)
+
+A tree representation of the abstract syntactic structure of source code.
+
+Each node of the tree denotes a construct occurring in the code. The syntax is
+abstracted away and does not represent every detail appearing in the syntax, but
+rather just the structural details. 
+
```
Comment (jstirnaman) [inline content/influxdb/v2.3/reference/glossary.md:23]: Is _order of execution_ worth mentioning?

Comment (jstirnaman) [inline content/influxdb/v2.3/reference/glossary.md:23]:
```suggestion

For more information about AST design, see [Abstract Syntax Tree in Wikipedia](https://en.wikipedia.org/wiki/Abstract_syntax_tree).
```

File: content/influxdb/v2.3/reference/glossary.md [hunk 2]
```diff
@@ -13,6 +13,14 @@ influxdb/v2.3/tags: [glossary]
...
 ### agent
 
 A background process started by (or on behalf of) a user and typically requires user input.
```
Comment (jstirnaman) [inline content/influxdb/v2.3/reference/glossary.md:26]:
```suggestion
A background process started by (or on behalf of) a user and typically requiring user input.
```

File: content/influxdb/v2.2/reference/glossary.md [hunk 3]
```diff
@@ -13,6 +13,14 @@ influxdb/v2.2/tags: [glossary]
...
+
+A tree representation of the abstract syntactic structure of source code.
+
+Each node of the tree denotes a construct occurring in the code. The syntax is
+abstracted away and does not represent every detail appearing in the syntax, but
+rather just the structural details.
+
 ### agent
 
 A background process started by (or on behalf of) a user and typically requires user input.
```
Comment (jstirnaman) [inline content/influxdb/v2.2/reference/glossary.md:26]:
```suggestion
A background process started by (or on behalf of) a user and typically requiring user input.
```

File: content/influxdb/v2.2/reference/glossary.md [hunk 4]
```diff
@@ -13,6 +13,14 @@ influxdb/v2.2/tags: [glossary]
...
 ## A
 
+### abstract syntax tree (AST)
+
+A tree representation of the abstract syntactic structure of source code.
+
+Each node of the tree denotes a construct occurring in the code. The syntax is
+abstracted away and does not represent every detail appearing in the syntax, but
+rather just the structural details.
+
```
Comment (jstirnaman) [inline content/influxdb/v2.2/reference/glossary.md:23]:
```suggestion

For more information about AST design, see [Abstract Syntax Tree in Wikipedia](https://en.wikipedia.org/wiki/Abstract_syntax_tree).
```

**Review by jstirnaman — APPROVED** [review]
A few suggestions.

File: content/influxdb/v2.2/reference/glossary.md (hunk 3 above)
Comment (kelseiv) [reply content/influxdb/v2.2/reference/glossary.md:26]: that typically requires user input?

**Pull request merged**